docs/basic/flattening.md

```diff
@@ -0,0 +1,134 @@
+# Flattening
+
+Flattening is the process of taking a raw AFM image, and removing the artefacts that are present in the image,
+resulting in an image where the background mica is flat and the sample is clearly visible resting on the surface.
+
+Here is a raw, unprocessed AFM image:
+
+![raw AFM image](../_static/images/flattening/flattening_raw_afm_image.png)
+
+You can see there is a heavy tilt in the image, as well as a lot of horizontal banding. These artefacts are removed
+during the flattening process in TopoStats.
+
+## TL;DR
+
+Images are processed by:
+
+- Row alignment (make each row median the same height)
+- Tilt & polynomial removal (fit a plane and quadratic polynomial to the image and subtract)
+- Scar removal (remove long, thin, bright streaks in the data)
+- Zero the average height (lower the image by its median height) to make the background roughly centred at zero nm
+- Masking (detect objects on the surface and flatten the image again, ignoring the data on the surface)
+- Secondary flattening (re-process the data using the mask to tell us where the background is)
+- Gaussian filter (to remove jitteryness / noise)
+
+![flattening pipeline](../_static/images/flattening/flattening_pipeline.png)
+
+## Row alignment
+
+The first step in the flattening process is **row alignment**. Row alignment is a process that adjusts the height of
+each row of the image so that they all share the same median height value. This gets rid of some of the horizontal
+banding. This leaves an image where the rows are aligned, but the image still has a clear tilt.
+
+![row alignment](../_static/images/flattening/flattening_align_rows.png)
+
+## Tilt removal
+
+After row alignment, tilt removal is applied. This is a simple process of fitting a plane to the image and subtracting
+it, resulting in a mostly flat image, however as you can see in the following image, it's not perfect.
+Two images are provided here, one with the full z-range and one with an adjusted z-range to show the remaining
+artefacts better.
+
+![tilt_removal_full_zrange](../_static/images/flattening/flattening_tilt_removal_full_zrange.png)
+
+![tilt removal_better_viewing](../_static/images/flattening/flattening_tilt_removal.png)
+
+## Polynomial removal
+
+After the tilt, we remove the polynomial trends. In some images, there is also quadratic or occasionally cubic bowing to
+the image too. We remove this by fitting a two dimensional quadratic polynomial to the image (in the horizontal
+direction), and subtracting it from the image. We then do the same for a nonlinear polynomial (z = a*x*y) to eliminate
+“saddle” trends in the data. We could do all of these at the same time, but we like to be able to see the iterative
+differences.
+
+## Scar removal
+
+We then run scar removal on the image. This is a special function that detects scars (long, thin, bright streaks in the
```

Comment:
Are these always bright? can it handle low streaks like in the image you've provided?